js/utility/calculateDuration.js

```diff
@@ -0,0 +1,10 @@
+import { msPerDay } from "/js/constants.js";
+export default function calculateDuration(taskStartDateString) {
```

Comment:
use line break to separate import/export statements.

```suggestion
import { msPerDay } from "/js/constants.js";

export default function calculateDuration(taskStartDateString) {
```

Reply:
done!

Reply:
done!